app/invitations/controllers.py

```diff
@@ -123,6 +125,56 @@ def send_request(new_user, committee):
         db.session.rollback()
         return Response.RequestError
 
+##
+## @brief      Sends an invitation to a committee-head to close
+##             a charge.
+##
+## @param      committee       The committee to join.
+## @param      new_user        The user to be invited.
+##
+## @return     True if email sent, False if not.
+##
+def send_close_request(user, committee, chargeID):
+    invite = and_(
+        Invitations.user_name == committee.head,
+        Invitations.committee_id == committee.id,
+        Invitations.isInvite == False
+    )
+
+    if Invitations.query.filter(invite).first() is not None:
+        return Response.InviteExists
+
+    invitation = Invitations (
+        user_name= user.id,
+        committee= committee,
+        committee_id = committee.id,
+        charge_id = chargeID,
+        isInvite=False
+    )
+
+    try:
+        db.session.add(invitation)
+        db.session.commit()
+        email = {}
+        email["title"] = "Close Charge Request"
+        email["sender"]=("SG TigerTracker", "sgnoreply@rit.edu")
+        email["recipients"] = [committee.head + "@rit.edu"]
```

Comment:
Related to my above comment but an admin should be the one receiving an email that a charge has been requested to be closed. You can retrieve admins from the database to be used as recipients.